src/SFA.DAS.EAS.Web/Orchestrators/EmployerManageApprenticeshipsOrchestrator.cs

```diff
@@ -627,6 +630,43 @@ private bool CanChangeDateStepBeSkipped(ChangeStatusType changeType, GetApprenti
             }, hashedAccountId, externalUserId);
         }
 
+        public async Task UpdateStopDate(string hashedAccountId, string hashedApprenticeshipId, ChangeStatusViewModel model, string externalUserId, string userName, string userEmail)
+        {
+            var accountId = _hashingService.DecodeValue(hashedAccountId);
+            var apprenticeshipId = _hashingService.DecodeValue(hashedApprenticeshipId);
+
+            _logger.Info(
+                $"Updating Apprenticeship stop date to {model.ChangeType}. AccountId: {accountId}, ApprenticeshipId: {apprenticeshipId}");
+
+            await CheckUserAuthorization(async () =>
+            {
+                var data =
+                    await
+                        _mediator.SendAsync(new GetApprenticeshipQueryRequest
+                        {
+                            AccountId = accountId,
+                            ApprenticeshipId = apprenticeshipId
+                        });
+
+                CheckApprenticeshipStateValidForChange(data.Apprenticeship);
+
+                await _mediator.SendAsync(new UpdateApprenticeshipStopDateCommand
+                {
+                    UserId = externalUserId,
+                    ApprenticeshipId = apprenticeshipId,
+                    EmployerAccountId = accountId,
+
+                    ChangeType = (Domain.Models.Apprenticeship.ChangeStatusType)model.ChangeType,
```

Comment:
What is this ChangeType property? The only thing we can do with this command is change the stop date?